app/react/portainer/account/AccountView/HelmRepositoryDatatable/HelmRepositoryDatatable.tsx

```diff
@@ -1,7 +1,7 @@
 import { useMemo } from 'react';
 
 import { useCurrentUser } from '@/react/hooks/useUser';
-import helm from '@/assets/ico/helm.svg?c';
+import helm from '@/assets/ico/vendor/helm.svg?c';
```

Comment:
does the new import path match the svg? The diff here still shows the svg in `app/assets/ico/helm.svg`

Reply:
there are 2 SVGs - the older one is in vendor and new one is in ico. Some places old is being used